test/framework/src/main/java/org/elasticsearch/test/XContentTestUtils.java

```diff
@@ -59,8 +59,10 @@ private static String differenceBetweenMapsIgnoringArrayOrder(String path, Map<S
             return path + ": sizes of the maps don't match: " + first.size() + " != " + second.size();
         }
 
-        for (String key : first.keySet()) {
-            String reason = differenceBetweenObjectsIgnoringArrayOrder(path + "/" + key, first.get(key), second.get(key));
+        for (Map.Entry<String, Object> clusterState : first.entrySet()) {
```

Comment:
Looking up all the usages of this method, there are 2 code paths that end up here, both times this object is a clusterstate.

Should I also rename them in the methods where they are passed along to this point in the code?